generic_command.py

```diff
@@ -0,0 +1,34 @@
+import asyncio
+
+import iterm2
+
+
+SCRIPT = 'SCRIPT'
+PREFIX = 'PREFIX'
+
+async def main(connection):
+    component = iterm2.StatusBarComponent(
+        short_description='Execute a user defined command',
+        detailed_description='A super simple component where both the command creating the text and a prefix can be defined by the user',
```

Comment:
Things aren't often as simple as we think 😉 

```suggestion
        detailed_description='A component that displays the output of an arbitrary command with an optional prefix',
```